.github/workflows/deploy-eb.yml

```diff
@@ -5,6 +5,8 @@ on:
       - staging
       - master
 
+# Making sure that the current deployment is completed before the next one
+concurrency: deploy_eb
```

Comment:
Might want to prefix this with the branch name `<staging | master>_deploy_eb`. So that staging deploys don't block production deploys.

Reply:
Oh yea lemme fix that